flink-tests/src/test/java/org/apache/flink/test/accumulators/AccumulatorErrorITCase.java

```diff
@@ -91,11 +91,9 @@ public void testFaultyAccumulator() throws Exception {
 		try {
 			env.execute();
 			fail("Should have failed.");
-		} catch (ProgramInvocationException e) {
-			Assert.assertTrue("Exception should be passed:",
-					e.getCause() instanceof JobExecutionException);
+		} catch (JobExecutionException e) {
```

Comment:
`ProgramInvocationException` are thrown during execution of the user program. `JobExecutionException` is thrown while the job is executing. That makes much more sense IMHO. The API doesn't strictly define any exceptions (`throws Exception`). Thus, we're not breaking.